app/models/agents/user_location_agent.rb

```diff
@@ -42,6 +44,7 @@ def default_options
         'secret' => SecureRandom.hex(7),
         'max_accuracy' => '',
         'min_distance' => '',
+        'api_key' => '',
```

Comment:
In the issue you mentioned that the api key is required, does that mean the agent can not function without it? If that is the case it would be good to check for the presence of that option. An awesome addition would be to actually validate the key against the google API, but I don't know if that is possible.